index.html

```diff
@@ -1,6 +1,57 @@
-<head>
-  <script type="text/javascript" src="https://firefoxux.github.io/SharedBanner/fxux-banner.min.js"></script>
```

Comment:
Hmmm.  I think we still want the banner at the top of the page.  But we should update that url to point to `design.firefox.com`. 🙂